code/modules/spells/targeted/cleric_spells.dm

```diff
@@ -110,3 +111,144 @@
 
 
 	return "You will now heal twice as much, but take twice as much damage. It will probably kill you."
+
+/spell/targeted/mend
+	name = "Mend Wounds"
+	desc = "This spell heals internal wounds and broken bones."
+	feedback = "MW"
+	spell_flags = INCLUDEUSER | SELECTABLE | NEEDSCLOTHES
+	invocation = "Ges'undh'eit"
```

Comment:
Thats the german word for health.
Gesundheit

Reply:
That is on puporse, a lot of wizard's invocations are references or 'jokes".